kafka/traits/apicurio-v0.0.1.yaml

```diff
@@ -0,0 +1,14 @@
+name: apicurio
```

Comment:
Can we include these examples inside the README.md file? Let's try to make the spec a single document as much as we can, so it's easier to work with it.

Reply:
The motivation (and I can't take the credit for it as it was @lbroudoux's idea, although I thought it was really good) for putting them in separate files was to enable people to refer to common sets of serdes headers as message traits.

e.g. https://gist.github.com/dalelane/3931c17b14c51fa4a1cf25496237d188#how-this-could-be-described-in-asyncapi-5 

```yaml
channels:
  my.topic.name:
    ...
    subscribe:
      ...
      message:
        schemaFormat: "application/vnd.apache.avro;version=1.9.0"
        contentType: "application/octet-stream"
        traits:
          - $ref: https://github.com/asyncapi/bindings/raw/master/kafka/traits/apicurio-v0.0.1.yaml
```

It makes a nice convenient short-hand to avoid having to copy/paste these standard serdes headers into many specs.

Reply:
🤔 Ok. Then let's put them inside an examples directory maybe? Sounds like they are part of the specification.

Reply:
I think this mechanism could also be used to integrate smoothly with CloudEvents.io. This will encourage building on and not limiting to this standard. Sometimes people tend to think that "if it's cloudevents.io compatible, it's sufficiently contractualized ..."

More generally, let's imagine having some "off-the-shelf" traits holding the headers for standards like CloudEvents and its different protocol bindings - but think also about OpenBanking async operations, Fiware, ....

These could be located into a dedicated repo where community can make PR contributing their own traits.

Reply:
@fmvilas @lbroudoux If we think that this better belongs in a different repo, do you think I should remove the examples from this PR, as I'd be reluctant to move them after people start referring to the URLs for trait definitions here?

Reply:
Removed these from this PR as I think we can consider them separately to the proposed changes to the bindings

Reply:
Sorry, I missed your previous comment. Yeah, I agree examples can be placed somewhere else. For now, we're putting them together at https://github.com/asyncapi/asyncapi/tree/master/examples/2.0.0.